CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v8.2.1](https://github.com/puppetlabs/puppetlabs-apache/tree/v8.2.1) (2022-09-26)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-apache/compare/v8.2.0...v8.2.1)
+
+### Fixed
+
+- \(maint\) Codebase Hardening [\#2313](https://github.com/puppetlabs/puppetlabs-apache/pull/2313) ([david22swan](https://github.com/david22swan))
```

Comment:
Given its size and potential impact I'd say this deserves to be called an enhancement rather than a bugfix, but I'll leave that up to you.

Reply:
It is a little on the border between the two, but think I would prefer leaving it as it is

Reply:
See https://github.com/puppetlabs/puppetlabs-apache/issues/2328. This requires Puppet 6.24. It should have updated the requirement:
https://github.com/puppetlabs/puppetlabs-apache/blob/a2be9bc4ea94179f33ae379f366f1d1661c64234/metadata.json#L86
That is also really at least a minor version bump, not a bugfix.